CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/gcp_api_util.py

```diff
@@ -24,7 +24,16 @@
 
 from google.cloud.pontem.sql import replicator
 
-DEFAULT_TIER = 'db-n1-standard-2'
+
+STORAGE_SCOPE = ['https://www.googleapis.com/auth/devstorage.read_write']
```

Comment:
STORAGE_SCOPE is a tad vague; CLOUDSQL_STORAGE or GCS_STORAGE_SCOPE etc?

Reply:
Changed to GCP_STORAGE_SCOPE.